### ISSUE TITLE
BetweenShowDelay=0 does not work

### ISSUE THREAD

**Issue by SamBent** [dotnet/wpf#4619]
* .NET Core Version: 3.0+
* Windows version: all
* Does the bug reproduce also in WPF for .NET Framework 4.8?: Yes
* Is this bug related specifically to tooling in Visual Studio (e.g. XAML Designer, Code editing, etc...)? no
* Security issues and bugs should be reported privately, learn more via our [responsible disclosure guidelines](https://github.com/dotnet/wpf/blob/master/README.md#reporting-security-issues-and-security-bugs).

 
 
 **Problem description:**
1. Run sample app from #4612
2. Move the mouse over the TextBox labeled "Tool tip"
3. Wait for the tooltip to display (5 sec)
4. Move the mouse down, away from the TextBox and into the surrounding GroupBox
 
 **Actual behavior:** 
The GroupBox's tooltip displays immediately

 **Expected behavior:**
The GroupBox's tooltip displays after a delay (5 sec)

The TextBox has BetweenShowDelay=0, meaning that the interval after closing its tooltip during which a new tooltip shows immediately, is empty (0 seconds).  WPF is not honoring this case.

**Comment by SamBent** [comment]
Fixed by the tooltip work in 6.0-preview7.
